docs/schema/querytransformparenttype-simpletype-sps15xsdsearchset4.md

```diff
@@ -7,7 +7,7 @@ ms.topic: reference
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 59305aac-635b-bcab-17d4-5ea82b108432
-description: "Last modified: March 09, 2015"
+description: "Describes the QueryTransformParentType simpleType (SPS15XSDSearchSet4). Applies to SharePoint Server 2013."
```

Comment:
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.